configurator/configurator-impl/src/main/java/org/codice/ddf/admin/configurator/impl/ConfigValidator.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.admin.configurator.impl;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+import org.apache.commons.collections.MapUtils;
+import org.apache.commons.lang.StringUtils;
+
+import com.google.common.io.Files;
+
+class ConfigValidator {
+    static void validateString(String bundleSymName, String msg) {
+        if (StringUtils.isBlank(bundleSymName)) {
+            throw new IllegalArgumentException(msg);
```

Comment:
This validator is package-only and currently used only by the `ConfiguratorImpl`. That service implementation is general purpose and already has its use semantics with values it returns. I could have left these three validator methods internal to that but for testing purposes it made sense to extract them here.

Callers should manage these validation exceptions (ideally they should never provide bad input) and can convert to message objects as needed.